common/slack/index.ts

```diff
@@ -0,0 +1,37 @@
+import { getLogger } from 'log4js';
+import { SlackBlock } from './blocks';
+
+export interface SlackChannel {
+    name: string;
+    webhookURL: string;
+}
+
+export type SlackMessage = { text: string } | { blocks: SlackBlock[] };
+
+export const SLACK_CHANNEL = {
+    TechAlerts: <SlackChannel>{
+        name: 'tech-alerts',
+        webhookURL: process.env.SLACK_TECH_ALERTS,
+    },
+    PublicStatistics: <SlackChannel>{
+        name: 'publ-statistics',
+        webhookURL: process.env.SLACK_PUBL_STATISTICS,
+    },
+};
+
+const logger = getLogger('Slack');
+
+export async function sendToSlack(channel: SlackChannel, message: SlackMessage) {
+    if (!channel.webhookURL) {
+        logger.warn('Cannot send message to ' + channel.name + ' as webhook is missing');
```

Comment:
We should try to not use variables in log messages. Otherwise, it's hard to search for all logs related to this event.